content/karpenter/050_karpenter/consolidation.md

```diff
@@ -288,4 +288,4 @@ In this section we have learned:
 
 * **Delete** Consolidation does also include events where instances are moved from `on-demand` to `spot`, however karpenter does not trigger **Replace** to make Spot node smaller as this can have an impact on the level of interruptions.
 
-* Karpenter uses cordon and drain [best practices](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) to terminate nodes. to make it safer, Karpenter adds a finalizer so that a `kubernetes delete node` command, results in a graceful termination that remove the node safely from the cluster.
\ No newline at end of file
+* Karpenter uses cordon and drain [best practices](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) to terminate nodes. to make it safer, Karpenter adds a finalizer so that a `kubernetes delete node` command, results in a graceful termination that remove the node safely from the cluster.
```

Comment:
I'm not sure what the new change is here

Reply:
Thanks, it was whitespace, I've removed and rebased to clean-up the git history.